src/InterventionBackend.php

```diff
@@ -340,6 +337,12 @@ public function loadFrom($path)
     public function setImageResource($image)
     {
         $this->image = $image;
+        if ($image === null) {
+            // remove our temp file if it exists
+            if (file_exists($this->getTempPath() ?? '')) {
+                unlink($this->getTempPath() ?? '');
```

Comment:
```suggestion
                unlink($this->getTempPath());
```
If the file exists, it can't be null. I suspect you copied this from `__destruct()` but for context, the null coalescing operator there was added by a naive automated approach to PHP 8.1 compatibility. We don't need or want to add unnecessary null coalescing operators in manual changes.

Reply:
Requested change made, and PR retargeted to 1.13